### PR TITLE
 [FEATURE] display content with data processors

### LINKED ISSUE #47: Finish with only one culomn, why ?
Under Typo3 10.4.10 (localhost with Xampp)- I followed the documentation step by step, 3 times and I always end with 1 single column instead of 3. Also I do not see the drop-down menu in the backend at the top of the page which gives the possibility to see the columns in a page. Where did I could have screw up?...And i may not be at the good place to ask that question, if so, where should i go ?

### PR DESCRIPTION
**Pull request by linawolf**
resolves #47, #31

### CODE REVIEW

File: Documentation/ContentMapping/Index.rst [hunk 1]
```diff
@@ -1,186 +1,372 @@
...
+                               }
+                           }
+                       }
+                   }
+               }
+           }
+       }
+   }
+
+After clearing the all caches the new backend layout is available in the page
```
Comment (jonaseberle) [inline Documentation/ContentMapping/Index.rst:87]: "the cache" - sorry, I have no deeper knowledge which and if a cache needs to be cleared for PageTS

File: Documentation/ContentMapping/Index.rst [hunk 2]
```diff
@@ -1,186 +1,372 @@
...
+main content area. It contains a headline (look for the :code:`<h2>`-tags) and
+some dummy content (look for the :code:`<p>`-tags).
 
 Simply replace these lines with the cObject-ViewHelper (`<f:cObject ... >`),
-so that file :file:`Default.html` shows the following HTML code. Make sure, you
-specify the column positions correctly (:code:`1`, :code:`0` and :code:`2`) and
-in exactly this order::
+so that file :file:`Default.html` shows the following HTML code.
+
+.. code-block:: html
```
Comment (jonaseberle) [inline Documentation/ContentMapping/Index.rst:220]: Maybe you want to use the short `<html>` opening tag from https://github.com/TYPO3/typo3/blob/master/typo3/sysext/fluid_styled_content/Resources/Private/Templates/Textmedia.html#L1?
That gives a sane place for custom ViewHelper namespaces later.

**Review by jonaseberle — APPROVED** [review]

File: Documentation/ContentMapping/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/ContentMapping/Index.rst:220]: I will replace all code examples by extracts form the extension anyway.

File: Documentation/ContentMapping/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/ContentMapping/Index.rst]: they do, these files get cached... I added an explainaition

**Pull request merged**